bundles/binding/org.openhab.binding.zwave/database/everspring/ad146.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>AD146</Model>
+	<Label lang="en">In-Wall Dimmer Module</Label>
+	<CommandClasses>
+		<Class><id>0x20</id></Class>
+		<Class><id>0x26</id></Class>
+		<Class><id>0x27</id></Class>
+		<Class><id>0x32</id></Class>
+		<Class><id>0x59</id></Class>		
+		<Class><id>0x5A</id></Class>		
+		<Class><id>0x5E</id></Class>		
+		<Class><id>0x70</id></Class>		
+		<Class><id>0x71</id></Class>		
+		<Class><id>0x72</id></Class>
+		<Class><id>0x73</id></Class>
+		<Class><id>0x7A</id></Class>
+		<Class><id>0x85</id></Class>
+		<Class><id>0x86</id></Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>short</Type>
+			<Minimum>0</Minimum>
+			<Maximum>255</Maximum>
+			<Default>255</Default>
+			<Size>2</Size>
```

Comment:
Are you sure the size is correct? It has a max value of 255 which just seems inconsistent with a 2 byte value (it still might be correct, but looks wrong, so just checking)